mne/io/base.py

```diff
@@ -1268,6 +1268,26 @@ def crop(self, tmin=0.0, tmax=None, copy=None):
         return raw
 
     @verbose
+    def segment(self, segment_length=1., verbose=None, **kwargs):
```

Comment:
I often used segmentation (back when I was using mostly matlab) to chop up the continuous signal (optionally drop epochs far away from task of interest) drop bad segments and calculate ICA on more data than just the epochs of interest. 
Or if I had to deal with data without epochs (like resting-state) segmentation was mostly for easy artifact rejection on continuous data.
But I'll be happy using `_segment_raw` function if you prefer to remove the `segment` method.